.github/workflows/email-petition-element.yml

```
@@ -0,0 +1,53 @@
+      - packages/email-petition-element/**
+      - packages/email-petition/**
+      - packages/petitions-components/**
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy email-petition-element
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)